app/components/bookings/create/create-booking.component.ts

```diff
@@ -1,4 +1,5 @@
 import {Component} from '@angular/core';
+import {TranslatePipe} from "ng2-translate/ng2-translate";
```

Comment:
@jdepoix  This file will be overwritten anyway after different feaature branches got merged into developement br